Shared/Level/Level.gd

```diff
@@ -88,3 +95,9 @@ func toggle_capture_points(toggle: bool) -> void:
 	Logger.info("Toggling capture points " + ("on" if toggle else "off") + ".", "capture_point")
 	for capture_point in _capture_points:
 		capture_point.active = toggle
+
+
+func set_team_id(team_id: int) -> void:
+	for capture_point in _capture_points:
+		if "player_team_id" in capture_point:
```

Comment:
Is there a case where this is false and not wrong?

Reply:
yes for the server capture point